docs/projects.md

```diff
@@ -4,36 +4,37 @@ A project is an app that contains React code and has a dependency on `react-nati
 
 Projects can provide additional properties to alter the CLI behavior, such as custom location of React Native files (this is useful when running RNTester from source) or a non-standard location of a project (useful when working in a brownfield app).
 
-## How does it work?
-
-A project can define a `react-native.config.js` at the root with custom configuration to be picked up by the CLI.
-
-For example, below configuration informs CLI of the additional assets to link and about a custom project location.
-
-```js
-module.exports = {
-  project: {
-    ios: {
-      project: './CustomProject.xcodeproj',
-    },
-  },
-  assets: ['./assets'],
-};
-```
-
-You can check all available options below.
-
-## Project interface
+## Interface
 
 ```ts
+interface Config = {
+  reactNativePath: string;
+  project: ProjectConfig;
+  dependencies: {[key: string]: Dependency};
+  platforms: {
+    android: PlatformConfig<
+      AndroidProjectConfig,
+      AndroidProjectParams,
+      AndroidDependencyConfig,
+      AndroidDependencyParams
+    >;
+    ios: PlatformConfig<
+      IOSProjectConfig,
+      IOSProjectParams,
+      IOSDependencyConfig,
+      IOSDependencyParams
+    >;
+    [name: string]: PlatformConfig<any, any, any, any>;
+  };
+  commands: Command[];
+}
 type ProjectConfigT = {
```

Comment:
This type is outdated. From the source:
```ts
export type ProjectConfig = {
  android?: AndroidProjectConfig;
  ios?: IOSProjectConfig;
  [key: string]: any;
};
```

Reply:
It's a mistake I have left it at all there. Was replaced by just "Config" to get rid of this confusion!